internal/resources/azure/data_factory_integration_runtime_managed.go

```diff
@@ -0,0 +1,103 @@
+}
+
+// DataFactoryIntegrationRuntimeManagedUsageSchema defines a list which represents
+// the usage schema of DataFactoryIntegrationRuntimeManaged.
+var DataFactoryIntegrationRuntimeManagedUsageSchema = []*schema.UsageItem{
+	{Key: "monthly_orchestration_runs", DefaultValue: 0, ValueType: schema.Int64},
+}
+
+// PopulateUsage parses the u schema.UsageData into the DataFactoryIntegrationRuntimeManaged.
+// It uses the `infracost_usage` struct tags to populate data into the DataFactoryIntegrationRuntimeManaged.
+func (r *DataFactoryIntegrationRuntimeManaged) PopulateUsage(u *schema.UsageData) {
+	resources.PopulateArgsWithUsage(r, u)
+}
+
+// BuildResource builds a schema.Resource from a valid DataFactoryIntegrationRuntimeManaged struct.
+// This method is called after the resource is initialised by an IaC provider.
+// See providers folder for more information.
+func (r *DataFactoryIntegrationRuntimeManaged) BuildResource() *schema.Resource {
+	runtimeFilter := "Azure Managed VNET"
+
+	costComponents := []*schema.CostComponent{
+		r.computeCostComponent(),
+		dataFactoryOrchestrationCostComponent(r.Region, runtimeFilter, r.MonthlyOrchestrationRuns),
+		dataFactoryDataMovementCostComponent(r.Region, runtimeFilter),
+		dataFactoryPipelineCostComponent(r.Region, runtimeFilter),
+		dataFactoryExternalPipelineCostComponent(r.Region, runtimeFilter),
+	}
+
+	return &schema.Resource{
+		Name:           r.Address,
+		UsageSchema:    DataFactoryIntegrationRuntimeManagedUsageSchema,
+		CostComponents: costComponents,
+	}
+}
+
+// computeCostComponent returns a cost component for cluster configuration.
+//
+// It is identical to SSIS runtime's cost component.
+func (r *DataFactoryIntegrationRuntimeManaged) computeCostComponent() *schema.CostComponent {
+	productType := "Standard"
+	if r.Enterprise {
+		productType = "Enterprise"
+	}
+
+	license := "License Included"
+	licenseTitle := " with license"
+	if !r.LicenseIncluded {
+		license = "AHB"
+		licenseTitle = ""
+	}
+
+	instanceTitle := "instance"
+	if r.Instances > 1 {
+		instanceTitle = "instances"
+	}
+
+	return &schema.CostComponent{
+		Name:           fmt.Sprintf("Compute (%s%s, %d %s)", productType, licenseTitle, r.Instances, instanceTitle),
+		Unit:           "hours",
+		UnitMultiplier: decimal.NewFromInt(1),
+		HourlyQuantity: decimalPtr(decimal.NewFromInt(r.Instances)),
+		ProductFilter: &schema.ProductFilter{
+			VendorName:    strPtr("azure"),
+			Region:        strPtr(r.Region),
+			Service:       strPtr("Azure Data Factory v2"),
+			ProductFamily: strPtr("Analytics"),
+			AttributeFilters: []*schema.AttributeFilter{
+				{Key: "meterName", ValueRegex: regexPtr(license)},
+				{Key: "skuName", ValueRegex: regexPtr(fmt.Sprintf("^%s$", r.InstanceType))},
+				{Key: "productName", ValueRegex: regexPtr(fmt.Sprintf("^SSIS %s", productType))},
+			},
+		},
+		PriceFilter: &schema.PriceFilter{
+			PurchaseOption: strPtr("Consumption"),
+		},
+	}
+}
```

Comment:
This resource is superseded by `azurerm_data_factory_integration_runtime_azure_ssis` in Terraform. It seems like they share the same instance types, but managed runtime also mentions other operations.